.github/workflows/minio-dotnet-linux.yml

```diff
@@ -16,14 +16,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        dotnet-version: ["6.0.x"]
+        dotnet-version: ["7.0.x"]
```

Comment:
```suggestion
        dotnet-version: ["6.0.x", "7.0.x"]
```

Reply:
The dotnet target version is still .NET6 only the MSBuild version used to build is .NET7. This is because i'm on a M1 Macbook and .NET6 and the older Resharper version turned out not to be compatible with ARM64. This would also make it easier to update to .NET7 in the future. I could also multi-target .net6 and .net7 in the nuget?

Reply:
Done.
6.0.x is added